dev/com.ibm.ws.org.apache.cxf.cxf.rt.frontend.jaxws.3.2/src/org/apache/cxf/jaxws/JAXWSMethodInvoker.java

```diff
@@ -94,6 +101,10 @@ protected Object invoke(Exchange exchange,
                 // and the chain is not suspended due to a continuation suspend
                 res = null;
                 changeToOneway(exchange);
+                params = Collections.singletonList(null);
```

Comment:
Is this liberty change? ( params = Collections.singletonList(null);),  is there a conflict with line 90?